### PR TITLE
README reworked

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,13 +13,13 @@ An unofficial, open source Android client for reddit.
...
 * Swipe posts left and right to perform customizable actions, such as upvote/downvote, or save/hide
-* Advanced cache management - automatically stores past versions of posts and comments
+* Advanced cache management—automatically stores past versions of posts and comments
 * Support for multiple accounts
-* Two-column tablet mode (can be used on your phone, if it’s big enough)
-* Image and comment precaching (optional: always, never, or Wi-Fi only)
+* Two-column tablet mode (can be used on your phone if it’s big enough)
+* Image and comment precaching (optional: always, never, or only on Wi-Fi)
 * Built-in image viewer, and GIF/video player
 * Multiple themes, including night mode, and ultra black for AMOLED displays
```
Comment (comradekingu) [inline README.md:24]: Don't remember it being called that in the strings.

File: README.md [hunk 2]
```diff
@@ -13,13 +13,13 @@ An unofficial, open source Android client for reddit.
...
-* Free and open-source Software - no ads/tracking
+* Copylefted libre software, licensed GPLv3+. Use, inspect, change and share at will; with all.
 * Lightweight and fast
 * Swipe posts left and right to perform customizable actions, such as upvote/downvote, or save/hide
-* Advanced cache management - automatically stores past versions of posts and comments
+* Advanced cache management—automatically stores past versions of posts and comments
 * Support for multiple accounts
-* Two-column tablet mode (can be used on your phone, if it’s big enough)
-* Image and comment precaching (optional: always, never, or Wi-Fi only)
+* Two-column tablet mode (can be used on your phone if it’s big enough)
```
Comment (comradekingu) [inline README.md:21]: Why not call it "Two-pane big screen mode" I think it said pane in the strings, but columns is probably better.

File: README.md [hunk 3]
```diff
@@ -54,15 +54,15 @@ https://hosted.weblate.org/projects/redreader/strings/
...
+RedReader is built using Gradle. On Linux, run:
 
     ./gradlew installDebug
 
 
 License
 -------
 
-RedReader is licensed under the GPL, version 3. A copy of the license is
+RedReader is licensed under the GPL version 3 or any later version. A copy of the license is
```
Comment (QuantumBadger) [inline README.md:65]: I can't retroactively relicense the app without getting permission from contributors ;)

**Comment by QuantumBadger** [comment]
Thanks for submitting this! I've had a look but to be honest I think I prefer most of the original wording. E.g. I think more people will understand what is meant by "free and open source" as opposed to "gratis and libre".

File: README.md (hunk 3 above)
Comment (comradekingu) [reply README.md:65]: Thought it already was. Not sure what the effective licensing is if it is neither specifically "only", and doesn't have "or any later version" in each file.

**Comment by comradekingu** [comment]
@QuantumBadger Not only does that not catch on, or get understood, it isn't "copyleft".
Making matters worse, the seemingly qualifying part "- no ads/tracking" has no bearing on it being "Free and open-source Software".
"Software" is curiously title-cased here, but far away from the "free" it is connected to.

**Pull request closed without merging**